routes/web.php

```diff
@@ -49,7 +49,7 @@
 })->middleware('guest:all');
 
 Route::get('/login', function () {
-    $all_kelas = Kelas::all();
+    $all_kelas = Kelas::where('id','!=',12)->orderBy('kelas','asc')->get();
```

Comment:
Nice catch @adhiiimf ! With this simple addition, we can prevent someone accidentally picked the `assistant's test drive classroom`

Reply:
Thankyou bang, and prevent praktikan become the impostor 🤣